Makefile

```diff
@@ -44,6 +45,7 @@ help:
 	@echo '   make serve [PORT=8000]           serve site at http://localhost:8000'
 	@echo '   make devserver [PORT=8000]       start/restart develop_server.sh    '
 	@echo '   make stopserver                  stop local server                  '
+	@echo '   make rsync_copy                  copy the website files via rsync   '
```

Comment:
Maybe instead be "copy the website files locally via rsync" (sorry to be so nit picky!)